IdentityCore/src/telemetry/request_telemetry/MSIDCurrentRequestTelemetrySerializedItem.m

```diff
@@ -64,7 +64,7 @@ - (NSString *)serialize
 {
     NSString *telemetryString = [NSString stringWithFormat:@"%@|%@|%@", self.schemaVersion, [self serializeFields: self.defaultFields], [self serializeFields: self.platformFields]];
     
-    if ([telemetryString lengthOfBytesUsingEncoding:NSUTF8StringEncoding] > 4000)
+    if ([telemetryString lengthOfBytesUsingEncoding:NSUTF8StringEncoding] > telemetryStringSizeLimit)
```

Comment:
Build failure due to comparison type mismatch can be fixed by changing this line :
if ((int)([telemetryString lengthOfBytesUsingEncoding:NSUTF8StringEncoding]) > telemetryStringSizeLimit)

Reply:
Thanks for pointing it out, its addressed